packages/pigeon/run_tests.sh

```diff
@@ -446,10 +445,12 @@ while getopts "t:l?h" opt; do
   esac
 done
 
+##############################################################################
```

Comment:
Could you file an issue for converting this file to Dart? There's a lot of logic here (as previously discussed).

Reply:
done https://github.com/flutter/flutter/issues/85068